index.html

```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
     "dateOfBirth": "1930-10-01"
   },
   "credentialSchema": {
-    "id": "https://api-test.ebsi.eu/trusted-schemas-registry/v2/schemas/z3MgUFUkb722uq4x3dv5yAJmnNmzDFeK5UC8x83QoeLJM",
+    "id": "https://code.europa.eu/ebsi/json-schema/-/raw/db67c771b96715c23ddf384abcfef33d6174d494/schemas/vcdm2.0/trust-model/schema.json",
     "type": "JsonSchema"
   },
   "termsOfUse": {
-    "id": "https://api-test.ebsi.eu/trusted-issuers-registry/v4/issuers/did:ebsi:zz7XsC9ixAXuZecoD9sZEM1/attributes/7201d95fef05f72667f5454c2192da2aa30d9e052eeddea7651b47718d6f31b0",
-    "type": "IssuanceCertificate"
+    "type": "TrustFrameworkPolicy",
```

Comment:
```suggestion
    "type": "AccreditationPolicy",
```

Reply:
Note that the EBSI documentation is still evolving and is internally inconsistent. For example 
https://hub.ebsi.eu/vc-framework/trust-model/issuer-trust-model-v4
says "the issued domain VC must contain a termsOfUse property with AttestationPolicy type"
whilst the schema does not contain this value.

Reply:
Hi @David-Chadwick. The schema https://code.europa.eu/ebsi/json-schema/-/blob/main/schemas/vcdm2.0/trust-model/schema.json?ref_type=heads is used for accreditations only (Root TAO and TAO)
The `AttestationPolicy` type is defined in the type extension https://code.europa.eu/ebsi/json-schema/-/tree/main/schemas/vcdm2.0/type-extensions/termsOfUse/AttestationPolicy?ref_type=heads What's missing in EBSI's documentation is how to resolve type extensions.